tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -158,13 +158,13 @@
 
     <!-- Integration: ServiceStack.Redis -->
     <!-- Assembly: ServiceStack.Redis -->
-    <!-- Latest package https://www.nuget.org/packages/ServiceStack.Redis/5.13.2 -->
-    <PackageReference Include="ServiceStack.Redis" Version="5.13.2" />
+    <!-- Latest package https://www.nuget.org/packages/ServiceStack.Redis/6.0.2 -->
+    <PackageReference Include="ServiceStack.Redis" Version="5.14.0" />
```

Comment:
The latest ServiceStack.Redis version is 6.0.2, but we aren't testing that due to the dependabot updater design. I wonder if the tests should explicitly _not_ have an upper bound, so that we _do_ test against versions we don't support, and hence lack-of-support is flagged more obviously than it currently is?

Reply:
Fixed in #2374, but we should probably still update the designer